components/Chips/src/MDCChipField.m

```diff
@@ -36,19 +36,19 @@
     MDCChipFieldVerticalInset, MDCChipFieldHorizontalInset, MDCChipFieldVerticalInset,
     MDCChipFieldHorizontalInset};
 
-@protocol MDCChipTextFieldDelegate <NSObject>
+@protocol MDCChipFieldTextFieldDelegate <NSObject>
 
 - (void)textFieldShouldRespondToDeleteBackward:(UITextField *)textField;
 
 @end
 
-@interface MDCChipTextField : MDCTextField
+@interface MDCChipFieldTextField : MDCTextField
 
-@property(nonatomic, weak) id<MDCChipTextFieldDelegate> deletionDelegate;
+@property(nonatomic, weak) id<MDCChipFieldTextFieldDelegate> deletionDelegate;
 
 @end
 
-@implementation MDCChipTextField
+@implementation MDCChipFieldTextField
```

Comment:
We should merge develop in to get the latest changes that already includes this rename